pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -2990,7 +2993,7 @@ public IonMobilityAndCCS GetIonMobility(ViewLibraryPepInfo pepInfo)
         {
             if (_selectedLibrary.TryGetIonMobilityInfos(new[] { pepInfo.Key }, out var ionMobilities))
             {
-                return ionMobilities.GetIonMobilityDict().Values.First().FirstOrDefault();
+                return ionMobilities.GetIonMobilityDict().Values.SelectMany(x => x).FirstOrDefault();

```

Comment:
Should this have ?? IonMobilityAndCCS.EMPTY; to maintain parity with the other return case where nothing is found? Is returning null hear meaningful? When you use FirstOrDefault() it seems to imply that you are going to sometimes return null.

Reply:
That would be better, yes.